styles.css

```diff
@@ -0,0 +1,600 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+v2.0 | 20110126
+License: none (public domain)
+*/
+
+html body div span applet object iframe h1 h2 h3 h4 h5 h6 p blockquote pre a abbr acronym address big cite code del dfn em img ins kbd q s samp small strike strong sub sup tt var b u i center dl dt dd ol ul li fieldset form label legend table caption tbody tfoot thead tr th td article aside canvas details embed figure figcaption footer header hgroup menu nav output ruby section summary time mark audio video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+
+article aside details figcaption figure footer header hgroup menu nav section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+p {
+  font-size: 22px;
+}
+
+small {
+  font-size: 16px;
+  color: #888;
+}
+
+/* Navigation bar */
+#navbar {
+  position: fixed;
+  width: 100%;
+  height: 50px;
+  display: flex;
+  background-color: white;
+  margin-top: -10px;
+}
+
+#header-flexbox {
+  display: flex;
+  justify-content: space-between;
+}
+
+#fficons {
+  padding-left: 20px;
+  width: 300px;
+  align-self: center;
+}
+
+#nytlogocont {
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  text-align: center;
+}
+
+#nytlogo {
+  width: 15%;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+}
+
+#buttons-navbarcont {
+  width: 400px;
+  align-self: center;
+}
+
+button {
+  background-color: rgb(86, 123, 149);
+  border: none;
+  color: white;
+  display: inline-block;
+  padding: 7px 10px 5px 10px;
+  border-radius: 3px;
+}
+
+/* Header */
+
+header {
+  padding-top: 50px;
+  text-align: center;
+  width: 40%;
+  padding-left: 30%;
+  padding-right: 30%;
+}
+
+#title {
+  display: inline-block;
+  font-size: 48px;
+  padding-top: 50px;
+}
+
+/* +++ Article +++ */
+
+.titlestyle {
+  font-weight: bold;
+  font-size: 30px;
+}
+
+.artpics1 {
+  display: grid;
+  grid-template-columns: 15% 70% 15%;
+}
+
+#one {
+  grid-column-start: 2;
+  max-width: 100%;
+  height: auto;
+}
+
+#onefig {
+  grid-column-start: 2;
+  color: #666;
+}
+
+#oneauth {
+  grid-column-start: 2;
+  display: inline-flex;
+}
+
+#dennis {
+  color: #333;
+}
+
+#denniso {
+  color: #333;
+}
```

Comment:
These selectors have the same style definition. Group selectors to minimize your code